pkg/routeagent_driver/handlers/ovn/handler.go

```diff
@@ -81,6 +83,8 @@ func (ovn *Handler) Init() error {
 		return err
 	}
 
+	ovn.startSyncer(ovn.stopCh)
```

Comment:
When I tried this PR locally, I see that we are invoking the sync APIs thrice. Any idea why?

```
2023-06-08T11:33:48.194Z INF ..ovn/config_syncer.go:89 OVN                  Waiting for interface to be ready: error adding host network routes: getNextHopOnK8sMgmtIntf returned error: could not find the route to [10.131.0.0/16] via "ovn-k8s-mp0"
2023-06-08T11:33:53.195Z DBG ../host_networking.go:130 OVN                  Processing route {Ifindex: 6 Dst: 10.131.0.0/16 Src: <nil> Gw: 10.131.2.1 Flags: [] Table: 254 Realm: 0}
2023-06-08T11:33:53.205Z DBG ../iptables/adapter.go:57 IPTables             In nat table, iptables rule "-j SUBMARINER-POSTROUTING", exists at index 1.
2023-06-08T11:33:53.205Z DBG ../iptables/adapter.go:78 IPTables             In nat table, iptables rule "-j SUBMARINER-POSTROUTING", already exists.
2023-06-08T11:33:53.206Z INF ..ovn/config_syncer.go:68 OVN                  Successfully synced route after "ovn-k8s-mp0" router delete on interface after retries
2023-06-08T11:33:53.206Z DBG ../host_networking.go:130 OVN                  Processing route {Ifindex: 6 Dst: 10.131.0.0/16 Src: <nil> Gw: 10.131.2.1 Flags: [] Table: 254 Realm: 0}
2023-06-08T11:33:53.212Z DBG ../iptables/adapter.go:57 IPTables             In nat table, iptables rule "-j SUBMARINER-POSTROUTING", exists at index 1.
2023-06-08T11:33:53.212Z DBG ../iptables/adapter.go:78 IPTables             In nat table, iptables rule "-j SUBMARINER-POSTROUTING", already exists.
2023-06-08T11:33:53.212Z INF ..ovn/config_syncer.go:68 OVN                  Successfully synced route after "ovn-k8s-mp0" router delete on interface after retries
2023-06-08T11:33:53.212Z DBG ../host_networking.go:130 OVN                  Processing route {Ifindex: 6 Dst: 10.131.0.0/16 Src: <nil> Gw: 10.131.2.1 Flags: [] Table: 254 Realm: 0}
2023-06-08T11:33:53.216Z DBG ../iptables/adapter.go:57 IPTables             In nat table, iptables rule "-j SUBMARINER-POSTROUTING", exists at index 1.
2023-06-08T11:33:53.217Z DBG ../iptables/adapter.go:78 IPTables             In nat table, iptables rule "-j SUBMARINER-POSTROUTING", already exists.
2023-06-08T11:33:53.217Z INF ..ovn/config_syncer.go:68 OVN                  Successfully synced route after "ovn-k8s-mp0" router delete on interface after retries
```

Reply:
But do we have to re-program for each event? Can we optimise the code?